qiskit/transpiler/passes/utils/remove_barriers.py

```diff
@@ -0,0 +1,48 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Remove all barriers in a circuit"""
+
+from qiskit.dagcircuit import DAGCircuit
+from qiskit.transpiler.basepasses import TransformationPass
+
+
+class RemoveBarriers(TransformationPass):
+    """Return a circuit with any barrier removed.
+
+    This transformation is not semantics preserving.
+
+    Example:
+
+        .. jupyter-execute::
+
+            from qiskit import QuantumCircuit
+            from qiskit.transpiler import PassManager
+            from qiskit.transpiler.passes import RemoveBarriers
+
+            circuit = QuantumCircuit(1)
+            circuit.x(0)
+            circuit.barrier()
+            circuit.h(0)
+
+            pm = PassManager(RemoveBarriers())
+            circuit = pm.run(circuit)
```

Comment:
You can call a pass directly, no need to build a pass manager.
```suggestion
            from qiskit import QuantumCircuit
            from qiskit.transpiler.passes import RemoveBarriers

            circuit = QuantumCircuit(1)
            circuit.x(0)
            circuit.barrier()
            circuit.h(0)

            circuit = RemoveBarriers()(circuit)
```